wasm/src/serialize.rs

```diff
@@ -0,0 +1,27 @@
+use crate::lazy_node::LazyNode;
+use clvmr::serde::{node_from_bytes, node_from_bytes_backrefs, serialized_length_from_bytes};
+use clvmr::{Allocator, ALLOW_BACKREFS};
+use std::rc::Rc;
+use wasm_bindgen::prelude::wasm_bindgen;
+
+#[wasm_bindgen]
+pub fn serialized_length(program: &[u8]) -> Result<u64, String> {
+    match serialized_length_from_bytes(program) {
+        Ok(length) => Ok(length),
+        Err(err) => Err(err.to_string()),
+    }
+}
+
+#[wasm_bindgen]
+pub fn sexp_from_bytes(b: &[u8], flag: u32) -> Result<LazyNode, String> {
+    let mut allocator = Allocator::new();
+    let deserializer = if (flag & ALLOW_BACKREFS) != 0 {
+        node_from_bytes_backrefs
+    } else {
+        node_from_bytes
+    };
+    match deserializer(&mut allocator, b) {
+        Ok(node) => Ok(LazyNode::new(Rc::new(allocator), node)),
+        Err(err) => Err(err.to_string()),
+    }
```

Comment:
same thing here. `map_err()` is probably what you want.

Reply:
Unlike the above, this code needs to map `Ok(NodePtr)` to `Ok(LazyNode)`
So I'll change the code as:
```rust
    let node = deserializer(&mut allocator, b).map_err(|e| e.to_string())?;
    Ok(LazyNode::new(Rc::new(allocator), node))
```